site/src/pages/DeploySettingsPage/AppearanceSettingsPage/AppearanceSettingsPageView.tsx

```diff
@@ -30,13 +31,17 @@ export type AppearanceSettingsPageViewProps = {
     preview: boolean,
   ) => void;
 };
+
+const fallbackBgColor = hslToHex(colors.blue[7]);
```

Comment:
KISS-related question: why do we need other changes in the PR? Did you identify any places where we can face HSL related issue?

Reply:
Yeah, KISS is definitely something I need to keep in mind more, but at least here, I feel like the defensive changes made sense.

The way I see it, there's three risks right now:
- `DashboardProvider` is already being a general-purpose component that can be imported by anything else
- The component is already being globally available to some degree because it's directly integrated into the `RequireAuth` component
- So many constants in the codebase are set up as HSL strings for readability. We could get rid of these in favor of hex strings, but they're a lot less intuitive to work with

I felt like having the provider be in charge of validating incoming string changes would be the best way to prevent any further mismatches from popping up

Reply:
Thanks for the explanation! Sounds good to me 👍